spiceaidocs/docs/components/data-connectors/mysql.md

```diff
@@ -71,3 +71,38 @@ datasets:
     params:
       mysql_connection_string: mysql://${secrets:my_user}:${secrets:my_password}@localhost:3306/my_db
 ```
+
+## Types
+
+The table below shows the MySQL data types supported, along with the type mapping to Apache Arrow types in Spice.
```

Comment:
Looks great, wondering if we should have special column for Timestamp with time zone?

Reply:
Thanks for pointing out! In fact MySQL doesn't have a timestamp with time zone type, and the timezone where timestamp is in is dependent on the timezone where mysql server is in. This should be documented as the expected property for mysql timestamp type mapping (i.e. arrow timestamp mapped from mysql will reflect whichever timezone mysql server is in)/